Modules/Messages/src/Messages.Domain/Entities/Message.cs

```diff
@@ -86,11 +86,13 @@ private void AnonymizeRecipient(IdentityAddress participantAddress, IdentityAddr
     {
         var recipient = Recipients.FirstOrDefault(r => r.Address == participantAddress);
         recipient?.UpdateAddress(anonymizedIdentityAddress);
+        RaiseDomainEvent(new MessageOrphanedDomainEvent(this));
     }
 
     private void AnonymizeSender(IdentityAddress anonymizedIdentityAddress)
     {
         CreatedBy = anonymizedIdentityAddress;
+        RaiseDomainEvent(new MessageOrphanedDomainEvent(this));
```

Comment:
Wtf. Why do you ALWAYS raise the `MessageOrphanedDomainEvent`, no matter if there are un-anonymized participants left?

Reply:
Changes made.